src/saltext/vmware/states/vmc_networks.py

```diff
@@ -0,0 +1,474 @@
+            subnets=subnets,
+            admin_state=admin_state,
+            description=description,
+            domain_name=domain_name,
+            tags=tags,
+            advanced_config=advanced_config,
+            l2_extension=l2_extension,
+            dhcp_config_path=dhcp_config_path,
+        )
+
+        if "error" in created_network:
+            return vmc_state._create_state_response(
+                name=name, comment=created_network["error"], result=False
+            )
+
+        return vmc_state._create_state_response(
+            name=name,
+            comment="Created Network {}".format(network_id),
+            new_state=created_network,
+            result=True,
+        )
+
+
+def absent(
+    name,
+    hostname,
+    refresh_key,
+    authorization_host,
+    org_id,
+    sddc_id,
+    verify_ssl=True,
+    cert=None,
+):
+    """
+    Ensure a given network/segment does not exist on given SDDC
+
+    name
+        Indicates the Network/segment id, any unique string identifying the network/segment.
+
+    hostname
+        The host name of NSX-T manager
+
+    refresh_key
+        API Token of the user which is used to get the Access Token required for VMC operations
+
+    authorization_host
+        Hostname of the VMC cloud console
+
+    org_id
+        The Id of organization to which the SDDC belongs to
+
+    sddc_id
+        The Id of SDDC from which the network/segment should be deleted
+
+    verify_ssl
+        (Optional) Option to enable/disable SSL verification. Enabled by default.
+        If set to False, the certificate validation is skipped.
+
+    cert
+        (Optional) Path to the SSL client certificate file to connect to VMC Cloud Console.
+        The certificate can be retrieved from browser.
+    """
+    network_id = name
+    log.info("Checking if Network with Id %s is present", network_id)
+    network = __salt__["vmc_networks.get_by_id"](
+        hostname=hostname,
+        refresh_key=refresh_key,
+        authorization_host=authorization_host,
+        org_id=org_id,
+        sddc_id=sddc_id,
+        network_id=network_id,
+        verify_ssl=verify_ssl,
+        cert=cert,
+    )
+
+    if "error" in network:
+        if "could not be found" in network["error"]:
+            network = None
+        else:
+            return vmc_state._create_state_response(
+                name=name, comment=network["error"], result=False
+            )
+
+    if __opts__.get("test"):
+        log.info("absent is called with test option")
+        if network:
+            return vmc_state._create_state_response(
+                name=name, comment="State absent will delete Network with Id {}".format(network_id)
+            )
+        else:
+            return vmc_state._create_state_response(
+                name=name,
+                comment="State absent will do nothing as no Network found with Id {}".format(
+                    network_id
+                ),
+            )
```

Comment:
**suggestion** adjust casing in these comments as well (i.e. capitalize only the first word, and `ID`)

Reply:
updated